cinema/views.py

```diff
@@ -87,6 +115,8 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         )
     )
     serializer_class = MovieSessionSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
this is a good approach, but if you use the same auth_classes or permissions_classes almost everywhere, you can set up default classes in REST_FRAMEWORK constant. Here are docs: [https://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy https://www.django-rest-framework.org/api-guide/authentication/#setting-the-authentication-scheme](https://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy%E2%80%A8https://www.django-rest-framework.org/api-guide/authentication/#setting-the-authentication-scheme)
If some view requires different classes, you can just override view attributes(like you do it right now)